packages/cf-component-progress/src/Progress.js

```diff
@@ -1,6 +1,95 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import Link from 'cf-component-link';
+import { createComponent } from 'cf-style-container';
+
+const styles = () => ({
+  position: 'relative'
+});
+
+const Bar = createComponent(
+  () => ({
+    display: 'block',
+    width: '100%',
+    appearance: 'none',
+
+    '&::-webkit-progress-bar': {
+      backgroundColor: '#ebebeb',
+      color: '#ebebeb'
+    },
+
+    '&::-webkit-progress-value': {
+      backgroundColor: '#2f7bbf',
+      color: '#2f7bbf',
+      transition: 'width 500ms ease'
+    },
+
+    /* Mozilla uses progress-bar as the value */
+    '&::-moz-progress-bar': {
+      backgroundColor: '#2f7bbf',
+      color: '#2f7bbf'
+    }
+  }),
+  'progress',
+  ['max', 'value']
+);
+
+const Items = createComponent(
+  () => ({
+    listStyle: 'none',
+    '&::after': {
+      content: '',
+      display: 'table',
+      clear: 'both'
+    },
+    margin: 0,
+    padding: 0
+  }),
+  'ol'
+);
+
+const Item = createComponent(
+  ({ theme, width, disabled, active }) => {
+    let color = theme.colorGray;
+    if (disabled) {
+      color = theme.colorGrayLight;
+    } else if (active) {
+      color = '#2f7bbf';
+    }
+
+    return {
+      width,
+      float: 'left',
+      height: 'auto',
+      padding: 0,
+      paddingTop: '7.5px',
+
+      color,
+      fontSize: '0.86667em',
+
+      textAlign: 'center',
+      cursor: disabled ? 'default' : 'pointer',
+
+      '&::before': {
+        content: 'none'
+      },
+
+      [`@media (max-width: ${theme.breakpoints.tablet})`]: {
+        display: active ? 'block' : 'none',
+        width: 'auto !important',
+        float: 'none'
+      },
+
+      '& > .cf-link': {
```

Comment:
This will not work when cf-component-link is migrated. The best solution here is to avoid the child selector, add some extra prop to Link as `isInProgressBar` and put these styles there.